app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { getTopics } = require("./controllers/news.controllers");
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use("*", (req, res) => {
+  res.status(404).send({ msg: "Invalid path" });
+});
+
```

Comment:
Consider adding a 500 internal error handler here as it will be useful in the long run 😁

Reply:
I added 500 status. Thank you!